src/workloads/scale/AuthNInsert.yml

```diff
@@ -1,7 +1,7 @@
 SchemaVersion: 2018-07-01
-Owner: "@mongodb/product-perf"
+Owner: "@mongodb/security"
 Description: |
-  TODO: TIG-3318
+  TODO: TIG-3318 This should be on security team. It needs a new ticket.
```

Comment:
Hey @spencerjackson, I think this workload is from your team, although it's older. I think Ben added it. Anyways, rather than me creating a new ticket and assigning it to you, if you or your team wanted to provide me a simple description for this workload, I could add it as part of this PR.

Reply:
FYI https://github.com/mongodb/genny/commit/810944b3419d6d84639f5edbfb49f1e86a74a21b#diff-4d91237b11aeaf1635bc21c3427d70bb69c51b0302c6b2e485ac3b67b375c8cb and TIG-1246

Another possible option would be to delete this workload. It is not run in regression and appears to be orphaned.

Reply:
How interesting... does this ensure that the server which this is run against is started with auth enabled? Or is this so orphaned that a bunch of work would be needed to make it function correctly? If "yes", we should delete this.

If this could work pretty easily,  a description like:
"This workload measures the cost of simple insert operations against servers which perform TLS encryption of wire protocol traffic and enforce authorization checks on each command invocation."

Reply:
I've optimistically added your description and added the autorun section to run against the ssl-replica configuration. I *think* this workload setups the client side to use SSL and auth, and the ssl-replica *should* have both setup, but eyes to verify that would be useful. I've setup a patch build and will link in the task when it runs (it's autogenerated). Here's the link for the replica-ssl config: https://github.com/10gen/dsi/blob/master/configurations/mongodb_setup/mongodb_setup.replica-ssl.yml and for comparison replica-auth: https://github.com/10gen/dsi/blob/master/configurations/mongodb_setup/mongodb_setup.replica-auth.yml
Hm, I don't see a `security` section in the replica-ssl, so maybe not.

Reply:
Here's the patch https://spruce.mongodb.com/task/sys_perf_linux_3_node_replSet_ssl_auth_n_insert_patch_8e681e80447e5859f79296e10911ffec39f31efa_61d887639ccd4e50f47dc23a_22_01_07_18_34_36/logs?execution=0 and it appears to fail
https://evergreen.mongodb.com/task_log_raw/sys_perf_linux_3_node_replSet_ssl_auth_n_insert_patch_8e681e80447e5859f79296e10911ffec39f31efa_61d887639ccd4e50f47dc23a_22_01_07_18_34_36/0?type=T#L1043

```[2022/01/07 20:34:42.258] 20:34:42Z>  [2022-01-07 20:34:42.226321] [0x00007fd233014700] [info]    Constructing pool with MongoURI 'mongodb://boss:password@10.2.0.200:27017/admin?appName=Genny&authMechanism=SCRAM-SHA-1&authSource=admin&replicaSet=rs0&ssl=true&tlsAllowInvalidHostnames=true'
 [2022/01/07 20:34:42.280] 20:34:42Z>  [2022-01-07 20:34:42.247793] [0x00007fd20caee700] [info]     Inserting { "a" : 55 }
 [2022/01/07 20:34:42.298] 20:34:42Z>  [2022-01-07 20:34:42.265800] [0x00007fd20caee700] [error]   Metrics not reported because operation 'Insert' being run by actor 'Inserter' did not close with success() or failure().
 [2022/01/07 20:34:42.298] 20:34:42Z>  [2022-01-07 20:34:42.265900] [0x00007fd20caee700] [error]   Unexpected std::exception: Authentication failed.: generic server error
 ```

@spencerjackson I'll just delete the file, unless you want to take a stab at it. Maybe we come back to this and others in our upcoming https://jira.mongodb.org/browse/PM-2184. Sound good?